downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -230,7 +230,7 @@ abstract class VersionControlSystem {
         /**
          * Return a simple string representation for the VCS this working tree belongs to.
          */
-        fun getType() = this@VersionControlSystem.toString()
+        fun getType() = this@VersionControlSystem.getType()
```

Comment:
Unrelated to this change, but this should also be a property: `val type = this@VersionControlSystem.type`